src/include/storage/store/rel_table.h

```diff
@@ -145,14 +125,16 @@ class DirectedRelTableData {
         vector<shared_ptr<ValueVector>>& outputVectors);
 
 private:
-    table_property_columns_map_t propertyColumns;
-    table_adj_columns_map_t adjColumns;
-    table_property_lists_map_t propertyLists;
-    table_adj_lists_map_t adjLists;
+    unordered_map<property_id_t, unique_ptr<Column>> propertyColumns;
+    unique_ptr<AdjColumn> adjColumn;
+    unordered_map<property_id_t, unique_ptr<Lists>> propertyLists;
+    unique_ptr<AdjLists> adjLists;
     table_id_t tableID;
+    table_id_t boundTableID;
     RelDirection direction;
     ListsUpdatesStore* listsUpdatesStore;
     BufferManager& bufferManager;
+    bool isSingleMultiplicityInDirection;
```

Comment:
Add a TODO for me to remove this eventually

Reply:
Added a todo